features/step_definitions/stepdefs.js

```diff
@@ -1,20 +1,26 @@
 const assert = require("assert");
 const { Given, When, Then } = require("@cucumber/cucumber");
 const axios = require("axios");
-const { checkUserExists } = require("../utils/apiUtils");
+const {
+  getUsersList,
+  checkUserExists,
+  createUser,
+  updateUser,
+  deleteUser,
+} = require("../utils/apiUtils");
 
 Given("check that the user with id {int} exists", async function (id) {
   const userExists = await checkUserExists(id);
   assert.ok(userExists);
 });
 
 When("a list of users is requested", async function () {
-  const resp = await axios.get("https://reqres.in/api/users");
+  const resp = await getUsersList();
   this.listOfUsers = resp;
 });
 
 When("a list of users on page {int} is requested", async function (page) {
-  const resp = await axios.get(`https://reqres.in/api/users?page=${page}`);
+  const resp = await await getUsersList(page);
```

Comment:
Not sure if we need await here, since getUsersList already has one.
`await await getUsersList(page);` -> `getUsersList(page);`

Same for others

Reply:
Also could be inlined
```js
this.listOfUsers = getUsersList(page)
```